pkg/validation/errors.go

```diff
@@ -111,6 +111,52 @@ func (e *InvalidComponentError) Error() string {
 	return fmt.Sprintf("the component %q is invalid - %s", e.componentName, e.reason)
 }
 
+//MissingProjectRemoteError returns an error if the git remotes object under a project is empty
+type MissingProjectRemoteError struct {
```

Comment:
This file continue to grow more and more. Maybe it worths having a separate file for each type of error: commands, projects...

Reply:
I'll keep this under consideration the next time the file is changed.  We will wind up with several files including a `common` one for an internal function under the same `validation` package  but if we have that many error files, it might make more sense to refactor these into an `error` package and export that function.